src/components/FormRegister.vue

```diff
@@ -0,0 +1,221 @@
+    <!-- Heading -->
+    <div class="q-my-lg">
+      <h1
+        class="text-h5 text-grey-10 text-bold q-my-none"
+        data-cy="form-register-title"
+      >
+        {{ $t('register.form.titleRegister') }}
+      </h1>
+    </div>
+    <!-- Form: register -->
+    <q-form @submit.prevent="onSubmitRegister">
+      <!-- Input: email -->
+      <div data-cy="form-register-email">
+        <!-- Label -->
+        <label for="form-register-email" class="text-caption text-bold">
+          {{ $t('register.form.labelEmail') }}
+        </label>
+        <!-- Input -->
+        <q-input
+          dense
+          outlined
+          v-model="formRegister.email"
+          lazy-rules
+          :rules="[
+            (val) => isValid(val) || $t('register.form.messageEmailReqired'),
+            (val) => isEmail(val) || $t('register.form.messageEmailInvalid'),
+          ]"
+          bg-color="grey-1"
+          id="form-register-email"
+          name="subject"
+          class="q-mt-sm"
+          data-cy="form-register-email-input"
+        />
+      </div>
+      <!-- Input: password -->
+      <div data-cy="form-register-password">
+        <!-- Label -->
+        <label for="form-register-password" class="text-caption text-bold">
+          {{ $t('register.form.labelPassword') }}
+        </label>
+        <!-- Input -->
+        <q-input
+          dense
+          outlined
+          hide-bottom-space
+          bg-color="grey-1"
+          v-model="formRegister.password"
+          id="form-register-password"
+          :hint="$t('register.form.hintPassword')"
+          :type="isPassword ? 'password' : 'text'"
+          :rules="[
+            (val) =>
+              isValid(val) || $t('register.form.messagePasswordRequired'),
+            (val) => isStrong(val) || $t('register.form.messagePasswordStrong'),
+          ]"
+          class="q-mt-sm"
+          data-cy="form-register-password-input"
+        >
+          <!-- Icon: show password -->
+          <template v-slot:append>
+            <q-icon
+              color="primary"
+              :name="isPassword ? 'visibility_off' : 'visibility'"
+              class="cursor-pointer"
+              size="18px"
+              @click="isPassword = !isPassword"
+              data-cy="form-register-password-icon"
+            />
+          </template>
+        </q-input>
+      </div>
+      <!-- Input: password confirm -->
+      <div class="q-mt-md" data-cy="form-register-password-confirm">
+        <!-- Label -->
+        <label
+          for="form-register-password-confirm"
+          class="text-caption text-bold"
+        >
+          {{ $t('register.form.labelPasswordConfirm') }}
+        </label>
+        <!-- Input -->
+        <q-input
+          dense
+          outlined
+          hide-bottom-space
+          bg-color="grey-1"
+          v-model="formRegister.passwordConfirm"
+          id="form-register-password"
+          :type="isPasswordConfirm ? 'password' : 'text'"
```

Comment:
Maybe we can use `lazy-rules` validation props.

Reply:
Added.